gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionExtension.groovy

```diff
@@ -117,4 +140,19 @@ class JavaServiceDistributionExtension extends BaseDistributionExtension {
     List<String> getExcludeFromVar() {
         return excludeFromVar
     }
+
+    void gc(String type, @Nullable Closure configuration) {
+        gc = objects.newInstance(profileNames[type])
```

Comment:
It looks like `plugins { ... } ` block doesn't add the plugin classpath to buildscript classpath so we can't reference classes directly, i.e. doing

```
gc (ResponseTime) { ... }
```

fails with MissingPropertyException `ResponseTime`. We could require use of classpath but that seemed a bit suboptimal. Happy to change this if there are better options.